src/Admin/Support.php

```diff
@@ -0,0 +1,187 @@
+<?php
+/**
+ * Class SupportMenu
+ *
+ * @package Ampproject\Ampwp
+ */
+
+namespace AmpProject\AmpWP\Admin;
+
+use AmpProject\AmpWP\Infrastructure\Conditional;
+use AmpProject\AmpWP\Infrastructure\Registerable;
+use AmpProject\AmpWP\Infrastructure\Service;
+use AmpProject\AmpWP\QueryVar;
+
+/**
+ * SupportMenu class.
+ */
+class Support implements Service, Conditional, Registerable {
+
+	/**
+	 * Check whether the conditional object is currently needed.
+	 *
+	 * @return bool Whether the conditional object is needed.
+	 */
+	public static function is_needed() {
+
+		return current_user_can( 'manage_options' );
+	}
+
+	/**
+	 * Adds hooks.
+	 */
+	public function register() {
+
+		/**
+		 * Add support link to Admin Bar.
+		 */
+		add_action( 'admin_bar_menu', [ $this, 'admin_bar_menu' ], 105 );
+
+		/**
+		 * Add support link to meta box.
+		 */
+		add_filter( 'amp_validated_url_status_actions', [ $this, 'amp_validated_url_status_actions' ], 10, 2 );
+
+		/**
+		 * Add support link to Post row actions.
+		 */
+		add_filter( 'post_row_actions', [ $this, 'post_row_actions' ], PHP_INT_MAX, 2 );
+
+		/**
+		 * Plugin row Support link.
+		 */
+		add_filter( 'plugin_row_meta', [ $this, 'plugin_row_meta' ], 10, 2 );
```

Comment:
No need for phpdoc-style comments. Simple one-line `//` comments are more appropriate.